shell.nix

```diff
@@ -107,6 +116,7 @@ haskell.project.shellFor {
 
   shellHook = ''
     ${pre-commit-check.shellHook}
+    ${deprecation-warning}
   '';
 
   # This is no longer set automatically as of more recent `haskell.nix` revisions,
```

Comment:
Is this in our new shell?

Reply:
Yes, it's the deprecation warning reminding the team to switch to `nix develop`

Reply:
No, I'm commenting on the bit about the locales.

Reply:
Ah, good catch, will look into it